src/lib/Form/Type/Search/SearchType.php

```diff
@@ -64,6 +65,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'placeholder' => /** @Desc("Any time") */ 'search.any_time',
                 'mapped' => false,
             ])
+            ->add(
+                'search_language',
+                ConfiguredLanguagesChoiceType::class,
+                [
+                    'required' => false,
```

Comment:
See my above comment. Here you could just use `choice_loader` option on `LanguageChoiceType` to avoid adding very specific `ConfiguredLanguagesChoiceType` class.